.github/actions/github-dependabot-check/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Dependabot check
+description: Checks if author is dependabot and if specific label is added to PR
+
+inputs:
+  gh-token:
+    description: 'GitHub token'
```

Comment:
Would be good to expand the description here explaining which permissions are expected for the provided GitHub token to work correctly in the context of this action.